src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -52,17 +53,17 @@ public void storeMessage(Message message) {
    * Gets messages posted by a specific user.
    *
    * @return a list of messages posted by the user, or empty list if user has never posted a
-   *     message. List is sorted by time descending.
-   *     Messages are public and will display on recipient's user page.
+   *     message. List is sorted by time descending. Messages are public and will display on
+   *     recipient's user page.
```

Comment:
This keeps popping back up to be with the other comment line, even when I try to manually fix it. Is this due to the styleguide? Should I try to disable this part, or is there a reason that Atom's styleguide or the Google plugin want the comment like this?

Reply:
I believe it is trying to fix the comment so the JavaDoc would include both sentences as part of the documentation for the return value. I think it is fine to just let it have its way so you don't have to fight with it later. This sentence seems a little out of place though since it is talking about high level display information in the Datastore code which shouldn't really care about how things are displayed.

Reply:
Okay, I will remove that sentence and merge, then, if you think that is best